mlflow/protos/service.proto

```diff
@@ -636,6 +636,9 @@ message RunInfo {
   // Unix timestamp of when the run ended in milliseconds.
   optional int64 end_time = 9;
 
+  // Unix timestamp of when the run is deleted in milliseconds.
+  optional int64 delete_time = 10;
+
```

Comment:
@Jason-CKY Rather than introducing a change to the `RunInfo` entity / proto, which has a larger change footprint that includes implementation work across several third-party MLflow services, can we start by adding the `delete_time` column / attribute to the SQLAlchemyStore / FileStore without modifying the proto? In the future, we may choose to extend the proto.

Reply:
I understand. we can start with small changes first. I changed the proto as it was causing some of the test cases to fail, but i'll revert the changes to protos!